InternetHospital.WebApi/Controllers/SignupController.cs

```diff
@@ -63,9 +77,17 @@ public async Task<ActionResult> Register([FromBody] UserRegistrationModel vm)
                     {
                         return BadRequest("Role type doesn't match conditions! must be only a Doctor or a patient");
                     }
+                    var userWithAvatar = await _uploadingFiles.UploadAvatar(file, user);
                     await _mailService.SendMsgToEmail(user.Email, "Confirm Your account, please",
                                  $"Confirm registration folowing the link: <a href='{callbackUrl}'>Confirm email NOW</a>");
-                    return Ok("Your account is created. Confirm your account on email!");
+                    if (userWithAvatar == null)
+                    {
+                        return Ok("Your account is created. But avatar didn't upload. Confirm your account on email!");
```

Comment:
Grammar mistake.
Change to:
"Your account is created. But avatar wasn't uploaded. Confirm your account on email!"